chainerx_cc/chainerx/stack_vector_test.cc

```diff
@@ -321,6 +321,7 @@ TEST(StackVectorTest, Back) {
 }
 
 TEST(StackVectorTest, Iterator) {
+#ifndef _WIN32
```

Comment:
Is it the case that all other tests pass? I think we should introduce a proper way of conditionally skipping tests and leave some failing, if that would be the case for Windows, for now.

Reply:
Added a comment
https://github.com/durswd/chainer/commit/b533a99346345bca410f0abeed4d58e787715594

It fails to compile. I don't understand why.

Reply:
Discussed offline that these tests can be removed (since they're comparing iterators with pointers).